xmppserver/src/main/java/org/jivesoftware/openfire/muc/MUCRoom.java

```diff
@@ -88,6 +86,12 @@ public class MUCRoom implements GroupEventListener, UserEventListener, Externali
         .setDynamic(true)
         .build();
 
+    public static final SystemProperty<Boolean> BULK_MSG_RETRACTION_ENABLED = SystemProperty.Builder.ofType(Boolean.class)
+        .setKey("xmpp.muc.bulkretraction")
```

Comment:
Please add an entry for `system_property.xmpp.muc.bulkretraction` in `openfire_i18n.properties` that is a single line, describing the purpose of this property.

Reply:
Done.